hugegraph-llm/src/hugegraph_llm/config/models/base_prompt_config.py

```
@@ -60,67 +68,73 @@ def ensure_yaml_file_exists(self):
+            return LiteralStr(val) if isinstance(val, str) else val
+        data = {
+            "graph_schema": to_literal(self.graph_schema),
+            "text2gql_graph_schema": to_literal(self.text2gql_graph_schema),
+            "extract_graph_prompt": to_literal(self.extract_graph_prompt),
+            "default_question": to_literal(self.default_question),
+            "custom_rerank_info": to_literal(self.custom_rerank_info),
+            "answer_prompt": to_literal(self.answer_prompt),
+            "keywords_extract_prompt": to_literal(self.keywords_extract_prompt),
+            "gremlin_generate_prompt": to_literal(self.gremlin_generate_prompt),
+            "doc_input_text": to_literal(self.doc_input_text),
+            "_language_generated": str(self.llm_settings.language).lower().strip(),
```

Pylint (3.3.7)

[error] Instance of 'BasePromptConfig' has no 'llm_settings' member

(E1101)